webapp/styles/Header.module.scss

```diff
@@ -53,7 +80,7 @@
 
 		.dropdownContainer {
 			padding: 9px 0 0 0;
-
+			z-index: 2;
```

Comment:
You can see that on opening any modal, the z-index makes the the container pops out with the modal. This is not desirable